### ISSUE TITLE
View path ($currentViewPath) should be relative to the project

### ISSUE THREAD

**Issue by marktopper** [serenohq/core#47]
In my `bade.blade.php` file I would like to add a extra menu item for everthing in the path `/docs/`.
How it be possible to have the path of the current rendering file as a variable?
Then I could do something like this.
```
@if (starts_with($path, '/docs/'))
    <li><a href="#">Link</a><li>
@endif
```

**Comment by znck** [comment]
`$currentViewPath` and `$currentUrlPath` are available in blade & markdown files.

**Comment by marktopper** [comment]
`$currentViewPath` does not start with `docs/` when using the `DocsExtension`.

`$currentUrlPath` is just what I needed. Thanks 👍

**Comment by znck** [comment]
Oh! `$currentViewPath` is relative path to directory. I guess relative to project makes more sense.

Reopen the issue if you want to change the behaviour of `$currentViewPath`.

**Comment by marktopper** [comment]
I must agree, it would give more sense to have the path to be relative to the project.
